web/src/javascript/drawMap.js

```diff
@@ -8,8 +8,8 @@ var yScreenPosOnMap = 200;
 
```

Comment:
Well, it's unreleated to this PR. So I'll let @Erikad94 do it if it's ok with you